scrapers/BlowBangGirls.yml

```diff
@@ -0,0 +1,30 @@
+name: BlowBangGirls
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - blowbanggirls.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //div[@class="videoDetails clear"]/h3
+     Date:
+        selector: //div[@class="videoInfo clear"]//span[contains(.,"Date Added:")]/following-sibling::text()
+        postProcess:
+          - parseDate: January 02, 2006
+     Performers:
+      Name: //span[@class="tour_update_models"]/a
+     Details: //div[@class="videoDetails clear"]/p
+     Tags:
+       Name: //span[@class="tour_update_tags"]/a
+     Image:
+        selector: //script[contains(text(),'/v3/content/')]/text()
+        postProcess:
+          - replace:
+              - regex: .+(?:poster=")([^"]*).+
+                with: https://www.blowbanggirls.com/$1
```

Comment:
In some scenes `https://www.blowbanggirls.com/v3/trailers/Vee.html` , `https://www.blowbanggirls.com/v3/trailers/Blowbang-Bukkake-Completely-Covered-in-Cum.html` , `https://www.blowbanggirls.com/v3/trailers/I-Want-Candee.html` the script was missing
I used the below
```yaml
      Image:
        selector: //script[contains(text(),'/v3/content/')]/text()|(//div[@class="player_thumbs"]/a)[1]/img/@src0_1x
        postProcess:
          - replace:
              - regex: .+(?:poster=")([^"]*).+
                with: $1
              - regex: ^
                with: https://www.blowbanggirls.com/
```
to get a fallback image